src-java/server42/server42-control-server-stub/src/main/java/org/openkilda/server42/control/serverstub/Server.java

```diff
@@ -0,0 +1,105 @@
+/* Copyright 2018 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.server42.control.serverstub;
+
+import org.openkilda.server42.control.messaging.flowrtt.Control.AddFlow;
+import org.openkilda.server42.control.messaging.flowrtt.Control.CommandPacket;
+import org.openkilda.server42.control.messaging.flowrtt.Control.CommandPacketResponse;
+import org.openkilda.server42.control.messaging.flowrtt.Control.CommandPacketResponse.Builder;
+import org.openkilda.server42.control.messaging.flowrtt.Control.Flow;
+import org.openkilda.server42.control.messaging.flowrtt.Control.PushSettings;
+import org.openkilda.server42.control.messaging.flowrtt.Control.RemoveFlow;
+
+import com.google.protobuf.Any;
+import com.google.protobuf.InvalidProtocolBufferException;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import org.zeromq.ZContext;
+import org.zeromq.ZMQ;
+import org.zeromq.ZMQ.Socket;
+
+import java.util.HashMap;
+import javax.annotation.PostConstruct;
+
+
+
+@Service
+@Slf4j
+public class Server extends Thread {
```

Comment:
could you please shared details of this server thread, javadoc will be ok as for me

Reply:
added, ping me if smt wrong.